README.md

```diff
@@ -10,9 +10,9 @@ the automatic deployment of quantum circuits on quantum hardware.
 Some of the key features of Qibolab are:
 
 - Deploy Qibo models on quantum hardware easily.
-- Create custom experimental drivers for custom lab setup.
+- Create experimental drivers for custom lab setup.
 - Support multiple heterogeneous platforms.
-- Use existing calibration procedures for experimentalists.
+- Use calibration procedures from [Qibocal](https://github.com/qiboteam/qibocal).
```

Comment:
Could you please port these changes to the sphinx docs too?

Reply:
Done in b5f8db4a169c81e35dcfb4437135efec81e767da.